server/src/main/java/org/opensearch/bootstrap/BootstrapCheck.java

```diff
@@ -32,18 +32,22 @@
 
 package org.opensearch.bootstrap;
 
+import org.opensearch.common.annotation.PublicApi;
+
 import java.util.Objects;
 
 /**
  * Encapsulates a bootstrap check.
  *
- * @opensearch.internal
+ * @opensearch.api
  */
+@PublicApi(since = "1.0.0")
```

Comment:
I think this can stay internal? What down streams depend on this not changing?

Reply:
Related to https://github.com/opensearch-project/OpenSearch/pull/9520#discussion_r1308986187